src/engraving/libmscore/edit.cpp

```diff
@@ -5153,6 +5153,9 @@ void Score::undoRemoveStaff(Staff* staff)
 
     staff->undoUnlink();
 
+    // If the staff we're removing is a "system object staff", it must be removed from the list
```

Comment:
the comment is, essentially, a duplication of what I can see from the code itself. Instead, I'd prefer to see explanation here of "why do we need to remove it here"

Reply:
Uhm, but I guess "why remove it here" is also self-explaining, given that the function is called "removeStaff"? Or did I misunderstand? Maybe I'd better just remove the comment

Reply:
yeah, that was just leading to unnecessary confusion